Resources/fsh-generated/resources/ValueSet-ISiKAccountIdentifierType.json

```diff
@@ -0,0 +1,25 @@
+{
+  "resourceType": "ValueSet",
+  "status": "draft",
+  "name": "ISiKAccountIdentifierType",
+  "id": "ISiKAccountIdentifierType",
+  "title": "ISiKAccountIdentifierType",
+  "version": "3.0.0-rc3",
+  "url": "https://gematik.de/fhir/isik/v3/Basismodul/ValueSet/ISiKAccountIdentifierType",
+  "experimental": false,
+  "publisher": "gematik GmbH",
+  "date": "2023-05-10",
+  "compose": {
+    "include": [
+      {
+        "system": "http://terminology.hl7.org/CodeSystem/v2-0203",
+        "concept": [
+          {
+            "code": "AN",
+            "display": "Account number"
+          }
```

Comment:
AN passt hier nicht. Gemäß https://terminology.hl7.org/5.1.0/CodeSystem-v2-0203.html ist die deutsche Übersetzung "Kontonummer", d. h. hier ist ein "bank account" gemeint. Da diese ganze in MII und ISiK vorgenommene Hierarchisierung und Klassifizierung von Account, Encounter und EpisodeOfCare so spezifisch auf das deutsche Gesundheitswesen zugeschnitten ist (zB sollte sich da ja auch eine FallAkte abbilden lassen), würde ich dringend dazu raten, hier ein eigenes, nationales Kodiersystem für Identifier zu definieren. Da könnte man dann analog zu dem sehr amerikanischen https://terminology.hl7.org/5.1.0/CodeSystem-v2-0203.html gleich auch SMC-B, TelematikID, GesundheitsID und andere spezifisch deutsche ID-Systeme mit reinpacken.

Reply:
@alexzautke , @patrick-werner , @simoneOnFhir, was sagt ihr dazu? Wir hatten uns im TC FHIR für die Basisprofile auf die Kodierung verständigt. 

1) Ich bin nicht überzeugt, dass hier bei AN "Account number" eine "Bankkontonummer" gemeint ist, sondern sehe das als Kodierung gemäß der Semantik des Account-Profils : "A financial tool for tracking value accrued for a particular purpose. In the healthcare field, used to track charges for a patient, cost centers, etc." . Unter diese Definition lässt sich unser Fall, der Identifier des Abrechnungsfalls, meiner Meinung nach weiterhin subsumieren.

2) Die deutsche Übersetzung von "Account Number" mit "Kontonummer", auf die @jcaumann hinweist, ist da meiner Meinung nach nicht optimal. -> vielleicht wäre das ein Thema für TC Terminologien (@patrick-werner )? Ggf. hier eine Änderung beantragen?

3) Trotzdem finde ich den Vorschlag von @jcaumann gut, dass wir ein eigenes CodeSystem für typisch deutsche Identifier erzeugen sollten. Die Frage ist nur, ob das nicht zu knapp ist für die Stufe 3 (die in wenigen Tagen in die Benehmensherstellung gehen soll). FYI @AnsgarHoeper

Reply:
issue für stufe 4 erzeugt. Sollte in Ag verfolgt werden unter Einbindung geeigneter Stakeholder (IDP ?  , KBV, DKG Hl7 International etc.)